suite-native/intl/src/en.ts

```diff
@@ -175,6 +175,18 @@ export const en = {
                 confirm: 'Continue with <type></type>',
             },
         },
+        coinDiscoveryRunningScreen: {
+            title: 'Checking {coin} for balances & transactions.',
+            subtitle: 'This should take just a moment.',
+        },
+        coinDiscoveryFinishedScreen: {
+            title: {
+                singular: 'We’ve found <count>1</count> <coin>Bitcoin</coin> account',
```

Comment:
nit: 
Does these "default values have any purpose here? What about omitting them? I have no strong opinion about this, rather thinking out loud.

```suggestion
                singular: 'We’ve found <count></count> <coin></coin> account',
```

Reply:
f471517693